### ISSUE TITLE
🐛 Outgoing synth packet is always "your" synth result even though its from "others"

### ISSUE THREAD

**Issue by WinterSolstice8** [DerpyProjectGroup/topaz#6545]
**_I affirm:_**
- [x] I understand that if I do not agree to the following points by completing the checkboxes my issue will be ignored.
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md) and the [Code of Conduct](https://github.com/LandSandBoat/server/blob/base/CODE_OF_CONDUCT.md).
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated.

## OS / platform the server is running (if known)

n/a

## Branch affected by issue 

`base`

## Steps to reproduce
enable filter of "other" synth results, see someone synth, get synth result anyway

Basically, we always send 0x06F when we should send 0x070 when it is received by "others". Our current implementation makes no distinction between "yours" and "others"

## Expected behavior

enable filter of "other" synth results, see someone synth, dont get their synth

**Comment by WinterSolstice8** [comment]
This issue was misinformed apparently -- see 
#6711

Will be closed by #6713
